docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
 
   django:
-    image: datasciencebr/jarbas-backend
+    image: jarbas
```

Comment:
Why not `image: datasciencebr/jarbas-backend`?

Reply:
sorry reverting

Reply:
See commit https://github.com/datasciencebr/jarbas/pull/285/commits/aa0d7aa9bff1f9bae3e7d8c3753932280c2ad61a